src/test/resources/de/retest/recheck/review/workers/recheck.ignore

```diff
@@ -2,6 +2,7 @@ matcher: id=banner
 matcher: id=title, attribute: font
 matcher: retestid=banner
 matcher: retestid=banner, attribute: outline
+matcher: retestid=banner, attribute: .*color
```

Comment:
What if i want to ignore a custom html-attribute: `data.theme` but not `data-theme`. We then would have to remember to escape this `data\.theme`, which, I think, is complex. 

I would rather use:
https://github.com/retest/recheck/pull/162#discussion_r271596388
with `$expression: regex=.*color`. Then it would be clear, that this needs escaping.

Regardless, we should be consistent with the `equals` or `matches` so that the user does not need to remember 1000 different use cases, but instead 5.

As proposed yesterday, do we need a workshop for this?

Reply:
I don't think a workshop is necessary. I'll just add Daniel as reviewer...